src/components/teampage/team-shared-styles.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components"
+import { royalBlue, navyBlue } from "../../constants/colors"
+import {
+  fontWeights,
+  HName,
+  P,
+  PCard,
+  Connect,
+} from "../../constants/typography"
+
+const MemberName = styled(HName)`
+  margin-bottom: 0;
```

Comment:
I think on laptop view, this margin can increase since it looks a little condensed